src/linear_algebra/std_basis.lean

```diff
@@ -13,11 +14,23 @@ elsewhere.
 
 To give a concrete example, `std_basis R (λ (i : fin 3), R) i 1` gives the `i`th unit basis vector
 in `R³`, and `pi.is_basis_fun` proves this is a basis over `fin 3 → R`.
+
+## Main definitions
+
+ - `std_basis R ϕ i b`: the `i`'th standard `R`-basis vector on `Π i, ϕ i`
+
+## Main results
+
+ - `is_basis_std_basis`: `std_basis` turns a componentwise basis into a basis on the product type
+ - `is_basis_fun`: `std_basis R (λ _, R) i 1` is a basis for `n → R`
```

Comment:
This should have the `pi` prefix so that it links

Reply:
```suggestion
 - `pi.is_basis_fun`: `std_basis R (λ _, R) i 1` is a basis for `n → R`
```